examples/website/point-cloud-ply/app.js

```diff
@@ -11,7 +11,7 @@ const FILE_PATH = 'examples/point-cloud-ply/lucy100k.ply';
 
 const INITIAL_VIEW_STATE = {
   lookAt: [0, 0, 0],
-  distance: 100,
+  distance: 2,
```

Comment:
there is a global camera projection change which I believe is a regression somewhere, for now let's change the distance to make this example work, need investigate where the real regression is.